src/js/search.js

```diff
@@ -51,19 +41,24 @@ class Search {
 		};
 
 		if (toggle) {
-			const clickHandler = function() {
+			const searchToggleClickHandler = function() {
 				if (transition) {
 					form.addEventListener(transition, transitionHandler);
 				} else {
 					setTimeout(transitionHandler, 300);
 				}
 
-				typeahead.hide();
+				headerEl.dispatchEvent(new CustomEvent('oHeader.searchToggle', {
+					bubbles: true,
+					detail: {
+						isOpen: getComputedStyle(form, null).getPropertyValue('visibility') === 'visible'
```

Comment:
IMHO it's a very silly that there is even a transition being used here (even sillier that when there's no transitions supported we add a 300ms delay!). Getting the visibility value once and passing it around would add complexity to the transition end handler, so I don't think it shoudl block this being released